src/style.css

```diff
@@ -1,3 +1,81 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
It's a good practice to provide fallback fonts in case the main one doesn't load. After `Roboto`, you could add a generic font family like `Arial` or `Helvetica`. (CSS checklist)